python/ml4ir/base/config/keys.py

```diff
@@ -135,7 +135,8 @@ class CalibrationKey(Key):
 
 class MonteCarloInferenceKey(Key):
     MONTE_CARLO_INFERENCE_TRIALS = "monte_carlo_inference_trials"
```

Comment:
@mohazahran you can change this to `MONTE_CARLO_TRIALS` as it is not limited to `inference`